hawkular-tenant-jaxrs-filter/pom.xml

```diff
@@ -64,8 +64,8 @@
 
   <dependencies>
     <dependency>
-      <groupId>javax</groupId>
-      <artifactId>javaee-api</artifactId>
+      <groupId>org.jboss.resteasy</groupId>
```

Comment:
Is there a reason to change this dependency? The `javax.javaee-api` should not bring any "extra" dependency that we don't already have provided by the AS.